xdg/openshot-qt.appdata.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
[Freedesktop says](https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html#spec-appdata-filespec) that the `"desktop"` type is legacy, and should be avoided in favor of `type="desktop-application"` for new AppData files.